setup.py

```diff
@@ -3,11 +3,14 @@
 except ImportError:
     from distutils.core import setup
 from pathlib import Path
+import re
 
 this_directory = Path(__file__).parent
 long_description = (this_directory / "README.md").read_text(encoding='utf8')
 
-print(long_description)
+# Replace relative paths to images with absolute paths
+long_description = re.sub("website/homepage/src/assets/img/", "https://raw.githubusercontent.com/gradio-app/gradio/main/website/homepage/src/assets/img/", long_description)
```

Comment:
In case we move around assets or the img folder, maybe replace at a higher level, e.g.`re.sub("website/homepage", "https://raw.githubusercontent.com/gradio-app/gradio/main/website/homepage"`